### DISCUSSION TITLE
Summer hiatus 2024

### DISCUSSION THREAD

**Discussion by timmbogner** [timmbogner/Farm-Data-Relay-System#232] [General]
I just want to make an official notice that I won't be around to work much on FDRS until the end of September.
I'll definitely hop back in if any issues arise. 
Edit Oct 8th:
Unfortunately, due to unexpected family circumstances it will be at least a week or two before I return. I'm looking forward to it!

**Comment by Stips5** [comment]
Mate have a nice rest, you did great job with this project so far, cant wait for new solutions feom you.

**Comment by timmbogner** [comment]
Updated original post
